specification/applicationinsights/resource-manager/microsoft.insights/stable/2015-05-01/componentWorkItemConfigs_API.json

```diff
@@ -0,0 +1,316 @@
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "An object containing the work item configuration definition created by this PUT call.",
+            "schema": {
+              "$ref": "#/definitions/WorkItemConfiguration"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WorkItemConfigurationsCreate": {
+            "$ref": "./examples/WorkItemConfigsCreate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/components/{resourceName}/DefaultWorkItemConfig": {
+      "get": {
+        "description": "Gets default work item configurations that exist for the application",
+        "operationId": "WorkItemConfigurations_GetDefault",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/WorkItemConfiguration"
+              }
+            }
+          },
+          "x-ms-examples": {
+            "WorkItemConfigurationsGetDefault": {
+              "$ref": "./examples/WorkItemConfigDefaultGet.json"
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": null
+          }
+        }
+      },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/components/{resourceName}/WorkItemConfigs/{workItemConfigId}": {
+      "delete": {
+        "description": "Delete an workitem configuration of an Application Insights component.",
+        "operationId": "WorkItemConfigurations_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkItemConfigIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The work item configuration that was successfully deleted.",
+            "schema": {              
+			  }
+          }
+        },
+        "x-ms-examples": {
+          "WorkItemConfigurationDelete": {
+            "$ref": "./examples/WorkItemConfigDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "WorkItemConfiguration": {
```

Comment:
Same comment about top level properties that are allowed.

Reply:
agreed, will change this in our next public version.